src/data/faq.json

```diff
@@ -2763,7 +2763,7 @@
                                 "active": false,
                                 "textblock": [
                                     "State of the art and well-established encryption mechanisms, for example using HTTP over TLS (HTTPS), ensure that messages are not readable for outside viewers. Metadata is removed before processing user data for the transmission of diagnosis keys. This prevents data from being linked together at database level.",
-                                    "Furthermore, various additional technical measures will be introduced shortly, for example, by randomly creating and sending false notifications that will be discarded on the server side. These false notifications cannot be distinguished from valid notifications and create a type of background noise. The transmission of TANs and upload of real infection data cannot be distinguished from this background noise. So, even with monitored network traffic, plausible deniability is still possible."
+                                    "Furthermore, various additional technical measures have been taken. For example, randomly creating and sending false notifications that will be discarded on the server side. These false notifications cannot be distinguished from valid notifications and create a type of background noise. The transmission of TANs and upload of real infection data cannot be distinguished from this background noise. So, even with monitored network traffic, plausible deniability is still possible."

```

Comment:
I think the "by" has to go back into this sentence, @MikeMcC399?

Reply:
@Ein-Tim 

The original English was a little weak grammatically and the changes carry this weakness over. I think it is good enough to publish as is though. If you want to make it better you could change the second sentence to:

"For example, false notifications that will be discarded on the server side are randomly created and sent."

This is a complete sentence and follows on directly as a detailed description of "additional technical measures"..

Reply:
Thanks for the improvement!